src/analytics/api/views/dashboard.py

```
@@ -189,17 +185,18 @@
         filter_value = json_data.get(filter_type)
 
         if filter_type in sites:
-            validated_value = filter_non_private_sites(filter_type, filter_value)
+            validated_value = filter_non_private_sites_devices(
+                filter_type, filter_value
+            )
         elif filter_type in devices:
-            validated_value = filter_non_private_devices(filter_type, filter_value)
+            validated_value = filter_non_private_sites_devices(
+                filter_type, filter_value
+            )
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)